pom.xml

```diff
@@ -795,7 +795,8 @@
                                 <fileSet>
                                     <directory>${jacoco.reportDir}</directory>
                                     <includes>
-                                        <File>**/*.exec</File>
+                                        <File>**/jacoco.exec</File>
+                                        <File>**/*-it.exec</File>
```

Comment:
Please restore the previous fileset.

Reply:
Practically, it's just a corner optimization to avoid multiple lines for the same test session in the final `jacoco.exec`. If that's OK, I'll happily remove it to get coverage, apply changes and potentialy CoS working again; I value the work hours spent on this issue too much to get them just nixed.

Please confirm.

Reply:
I don’t think it is a huge problem having multiple copies in the output.  It won’t increase the coverage reported.

Reply:
ACK, reverted  d7816a6